api/plan.go

```diff
@@ -163,19 +169,20 @@ func (e FeatureType) minimumSupportedPlan() PlanType {
 
 // FeatureMatrix is a map from the a particular feature to the respective enablement of a particular plan.
 var FeatureMatrix = map[FeatureType][3]bool{
-	"bb.feature.schema-drift":       {false, true, true},
-	"bb.feature.task-schedule-time": {false, true, true},
-	"bb.feature.multi-tenancy":      {false, true, true},
-	"bb.feature.dba-workflow":       {false, false, true},
-	"bb.feature.data-source":        {false, false, false},
-	"bb.feature.ghost":              {false, true, true},
-	"bb.feature.pitr":               {false, true, true},
-	"bb.feature.approval-policy":    {false, true, true},
-	"bb.feature.backup-policy":      {false, true, true},
-	"bb.feature.sql-review":         {false, true, true},
-	"bb.feature.rbac":               {false, true, true},
-	"bb.feature.3rd-party-auth":     {false, true, true},
-	"bb.feature.branding":           {false, true, true},
+	FeatureSchemaDrift:           {false, true, true},
+	FeatureTaskScheduleTime:      {false, true, true},
+	FeatureMultiTenancy:          {false, true, true},
+	FeatureDBAWorkflow:           {false, false, true},
+	FeatureDataSource:            {false, false, false},
+	FeatureGhost:                 {false, true, true},
+	FeaturePITR:                  {false, true, true},
+	FeatureApprovalPolicy:        {false, true, true},
+	FeatureBackupPolicy:          {false, true, true},
+	FeatureSQLReviewPolicy:       {false, true, true},
+	FeatureRBAC:                  {false, true, true},
+	Feature3rdPartyAuth:          {false, true, true},
+	FeatureBranding:              {false, true, true},
+	FeatureEnvironmentTierPolicy: {false, true, true},
```

Comment:
false, false, true

Reply:
@LiuJi-Jim FYI

Reply:
LGTM

Reply:
Updated.
@LiuJi-Jim You may also need to change that on the frontend.